### PR TITLE
projects page and some other updates

### PR DESCRIPTION
**Pull request by idil-bil**
-added the subtext to the title section of the about page
-added the team photo on the about page (looks like a blank box)
-fixed the alignment of the footer
-completed the projects page
-updated the title photo on join us
-organized the js code for join us

### CODE REVIEW

File: src/components/pages/Projects.js [hunk 1]
```diff
@@ -1,12 +1,99 @@
...
+                <p>- Metal (Aluminum, Brass, Bronze, Stainless Steel) <br /><br />
+                    - Carbon fiber <br /><br />
+                    - Flexible (TPU, TPE) <br /><br />
+                    - Polycarbonate <br /><br />
+                    - PTFE (Nylon) <br /><br />
+                    - Water soluble <br /><br />
+                    - PETG <br /><br />
+                    - Conductive <br /><br />
+                    - Magnetic <br /><br />
+                    - Wax and many more!
```
Comment (Abeilles14) [inline src/components/pages/Projects.js:53]: To create lists in html, use ul (unordered lists) for bullet points, and ol (ordered lists) for numbered lists. See here for more information: https://www.w3schools.com/html/html_lists.asp

File: src/components/pages/Projects.js [hunk 2]
```diff
@@ -1,12 +1,99 @@
...
     return (
         <>
             <h1 className='projects'>PROJECTS</h1>
-            <Cards />
+            <div className='info_wrapper'>
+                <h5>CONSULTING AND 3D PRINTING</h5>
+                <p>Our main and most popular team activity! Starting at only $0.15/gram and a  
+                    $5 setup fee, our printing service is one of the most affordable on  campus! 
+                    Using our 6 available printers, we aim to offer low-cost, high quality 3D 
+                    printing services to UBC students, staff, and faculty. <br /><br />
```
Comment (Abeilles14) [inline src/components/pages/Projects.js:14]: The correct syntax for breaks is <br> (the opening tag) and </br> (the closing tag).

File: src/components/pages/Projects.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    text-align: left;
+    font-family: Bahnschrift;
+    font-size: 14px;
+    padding: 15px 15px;
+    font-weight: lighter;
+    margin-left: 90px;
+    margin-right: 90px;
+    line-height: 20px;
+    box-sizing:border-box;
+  }
```
Comment (Abeilles14) [inline src/components/pages/Projects.css:22]: A side note on this (not super important but cleaner code): Notice that many of these properties are repeated. Let's see if we can combine them to make the code simpler and avoid repetition in the CSS attributes.

File: src/components/pages/Projects.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    font-family: Bahnschrift;
+    font-size: 15px;
+    font-weight: bold;
+    padding: 0px 15px 15px;
+    margin-left: 90px;
+    margin-right: 90px;
+    display: inline-block;
+  }
+
+.info_wrapper > p {
```
Comment (Abeilles14) [inline src/components/pages/Projects.css:12]: One worrying thing that I noticed and didn't know about before: these class attributes seem to be affected by other CSS classes with the same name from other CSS files (eg. JoinUs.css). From now on, let's make sure that each class has a distinct name.

**Comment by Abeilles14** [comment]
Think I fixed most of the issues mentioned above, just review and understand the comments and new changes just in case.

As our next step for the project page (Not in this PR), I think it might look better if we were to have cards or maybe 2 columns kind of structure like the consulting page, where each image is beside its text rather than under.
Please create a new issue for that and put it in the backlog on the Zenhub board!

image

File: src/components/pages/Projects.js (hunk 2 above)
Comment (idil-bil) [reply src/components/pages/Projects.js]: I tried changing the breaks but both <br></br> and just using <br /> did not work

**Comment by idil-bil** [comment]
One more problem I realized on the "Join us" page, the link for the application form is not formatted the same as "The application deadline is September 17th 2021." despite them both being defined as h3 but I don't know how to fix it

**Pull request merged**